.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
No actual change in this guy, just reformatting from `.eslintrc` to `.eslintrc.js` b/c javascript > json :wink: